src/content/contributing/translation-program/index.md

```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
 
 - [Content versions](/contributing/translation-program/content-versions/) _– Which pages are included in each content version of ethereum.org_
 
+## Translation Program mission and vision {#mission-and-vision}
```

Comment:
```suggestion
## Mission and vision {#mission-and-vision}
```

Reply:
Should this be the top section on the page (i.e. in front of "How to translate" & "Resources")?